products/oracle-database.md

```diff
@@ -0,0 +1,118 @@
+    eol: 2024-04-30
+    # The first year of extended support is free.
+    extendedSupport: 2027-04-30
+    link: https://docs.oracle.com/en/database/oracle/oracle-database/19/whats-new.html
+
+-   releaseCycle: "18c"
+    lts: false
+    releaseDate: 2018-07-23
+    eol: 2021-06-30
+    extendedSupport: false
+    link: https://docs.oracle.com/en/database/oracle/oracle-database/18/whats-new.html
+
+-   releaseCycle: "12cR2"
+    releaseLabel: "12c Release 2"
+    lts: false
+    releaseDate: 2017-03-01
+    eol: 2022-03-31
+    extendedSupport: false
+    link: https://docs.oracle.com/en/database/oracle/oracle-database/12.2/whats-new.html
+
+-   releaseCycle: "12cR1"
+    releaseLabel: "12c Release 1"
+    lts: true
+    releaseDate: 2013-06-25
+    eol: 2018-07-31
+    extendedSupport: 2022-07-31
+    link: https://docs.oracle.com/database/121/index.htm
+
+-   releaseCycle: "11gR2"
+    releaseLabel: "11g Release 2"
+    lts: true
+    # https://www.orafaq.com/wiki/Oracle_11gR2
+    releaseDate: 2009-09-01
+    eol: 2015-01-31
+    extendedSupport: 2020-12-31
+    link: https://docs.oracle.com/cd/E11882_01/index.htm
+
+-   releaseCycle: "11gR1"
+    releaseLabel: "11g Release 1"
+    lts: true
+    # https://www.orafaq.com/wiki/Oracle_11gR1
+    releaseDate: 2007-08-09
+    # https://web.archive.org/web/20190516170139/https://support.oracle.com/knowledge/Oracle%20Database%20Products/742060_1.html#foot1
+    eol: 2012-08-31
+    extendedSupport: 2015-08-31
+    link: https://docs.oracle.com/cd/B28359_01/index.htm
+
+-   releaseCycle: "10gR2"
+    releaseLabel: "10g Release 2"
+    lts: true
+    # https://web.archive.org/web/20180405025608/http://www.oracle.com/us/corporate/press/017324_EN
+    releaseDate: 2005-07-11
+    # https://web.archive.org/web/20190516170139/https://support.oracle.com/knowledge/Oracle%20Database%20Products/742060_1.html
+    eol: 2010-07-31
+    extendedSupport: 2015-07-31
+    link: https://docs.oracle.com/cd/B19306_01/nav/portal_1.htm
+
+-   releaseCycle: "10gR1"
+    releaseCycle: "10g Release 1"
+    lts: true
+    # https://www.orafaq.com/wiki/2003
+    releaseDate: 2003-09-08
+    # https://www.orafaq.com/wiki/Oracle_10g
+    eol: 2009-01-31
+    extendedSupport: 2012-01-01
+    link: https://www.oracle.com/database/technologies/database10g-doc.html
+
+---
+
+> Oracle Database is a commercial relational database management system (RDBMS) developed by Oracle
+> Corporation. It uses the Structured Query Language (SQL) for data manipulation and management, and
+> it provides features such as robust security, transactional consistency, and high availability.
+
+Oracle Database is a complex product distributed in various flavor (multiple cloud and on-premises
+offers), with support for various operating systems (such as Linux, Oracle Solaris or Microsoft
+Windows), and with [various support stages](https://www.oracle.com/support/lifetime-support/software.html "Lifetime Support for Oracle software")
```

Comment:
Mention https://support.oracle.com/knowledge/Oracle%20Cloud/161818_1.html. To access this document, an Oracle account with appropriate permissions is needed.